lib/mergeIndDisplay.ml

```diff
@@ -95,11 +96,20 @@ let print_differences conf base branches p1 p2 =
       match Image.get_portrait conf base p with
       | Some (`Url url) ->
           ({|<img src="|} ^<^ escape_html url
-           ^>^ {|" style="max-width:75px;max-height:100px">|}
+           ^>^ {|" style="max-width:75px;max-height:100px">|} ^ url
```

Comment:
Why do we apply `escape_html` to `url`?

This print the url as text next to the image?

Reply:
We can probably suppress this escape_url.
The same comment applies to `string_of_image_url` in perso.ml